src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableStore.kt

```diff
@@ -77,6 +78,13 @@ class VariableStore(
    */
   private val replacements = ConcurrentHashMap<VariableId, List<VariableId>>()
 
+  fun fetchDeliverableVariableIds(deliverableId: DeliverableId): List<VariableId> =
```

Comment:
I wasn't able to figure out what permissions this should require. We don't really have any "able to view X deliverable permissions", which would seem appropriate here. 

In the PRDs there are permissions related to the _values_ of the variables, which we will fetch in a subsequent PR. But seeing that those permissions are basically wide open, I thought it would be OK to leave this permission-less. Let me know if that assumption is incorrect.

Reply:
Yeah, I think right now we kind of consider deliverables to be public information. I could see maybe restricting them to people in accelerator-tagged orgs (and internal users, of course) but that seems out of scope for this change, if it's even needed at all.